src/server.js

```diff
@@ -1,6 +1,14 @@
+const MAX_ALLOWED_SESSION_DURATION = 14400;
+const twilioAccountSid = process.env.TWILIO_ACCOUNT_SID;
+const twilioApiKeySID = process.env.TWILIO_API_KEY_SID;
+const twilioApiKeySecret = process.env.TWILIO_API_KEY_SECRET;
+const twilioAuthToken = process.env.TWILIO_AUTH_TOKEN;
+const twilioMobileNumebr = process.env.TWILIO_MOBILE_NUMBER;
```

Comment:
💅 Small typo, should be:

```suggestion
const twilioMobileNumber = process.env.TWILIO_MOBILE_NUMBER;
```